cocos/terrain/terrain.ts

```diff
@@ -397,7 +397,7 @@ export class TerrainBlock {
         ];
 
         this._renderable._meshData = new RenderingSubMesh([vertexBuffer], gfxAttributes,
-            PrimitiveMode.TRIANGLE_LIST, this._terrain._getSharedIndexBuffer());
+            PrimitiveMode.TRIANGLE_LIST, this._terrain._createSharedIndexBuffer());
```

Comment:
use newly created index buffer might increase memory usage.
I think we need to modify rendering sub mesh and add a flag to rendering submesh.
if the rendering submesh owning the index buffer, then it needs to destroy the buffer.
if the rendering submesh not owning the index buffer, then it should not destroy the buffer

Reply:
when creating the RenderingSubMesh, we might add a new parameter indicating whether the RenderingSubMesh owning the index buffer.
This is a temporary solution.